src/utils/parse-file-list.js

```diff
@@ -33,6 +46,9 @@ async function convertDirsToGlobs (fileList, baseDir) {
 
             if (fileStat.isDirectory())
                 return path.join(file, TEST_FILE_GLOB_PATTERN);
+
+            if (OS.win)
+                file = modifyFileRoot(baseDir, file);
```

Comment:
It looks to complex.
I was be able to fix the problem using the following code:
```js
const absPath = path.isAbsolute(file) ? file : path.resolve(baseDir, file);
```

Reply:
We can simplify `modifyFileRoot (baseDir, file)` as follows:

```js
function modifyFileRoot (baseDir, file) { 
    const absPath = path.isAbsolute(file) ? file : path.resolve(baseDir, file);

   return baseDir[0] + absPath.slice(1);
}
```

@kirovboris will we do this or stay it as is?

Reply:
It looks like it doesn't work in the following case:
```bash
node C:\...\node_modules\testcafe\lib\cli\index.js chrome D:\...\test.js
```
We will try to run `C:\...\test.js` instead. But I agree the current implementation doesn't look very readable.

Reply:
I doubt if we should compare only the drive letter in the case-insensitive way.  E.g. I can mistype and use an uppercase or lowercase letter in any path fragment on Windows, e.g.
```
.\node_modules\.bin\testcafe chrome D:\test\release\test.js
.\node_modules\.bin\testcafe chrome D:\Test\release\test.js
.\node_modules\.bin\testcafe chrome D:\TEST\release\test.js
```

will run the same `test.js` on Windows, but will break test run detection.

Reply:
Sure, it makes sense, just the error with disk letter is more common.